Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -0,0 +1,77 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class ManagePledgeViewController: UIViewController {
+  private lazy var closeButton: UIBarButtonItem = {
+    UIBarButtonItem(
+      image: UIImage(named: "icon--cross"),
+      style: .plain,
+      target: self,
+      action: #selector(RewardsCollectionViewController.closeButtonTapped)
```

Comment:
Looks like this is referencing wrong controller (unless I'm missing something)...should this be `ManagePledgeViewController.close....`?

Reply:
Copy pasta 😬